### ISSUE TITLE
Firefly III Data Import Tool : Status code is 404. Body is: { "message": "The route api/v1/about could not be found." }

### ISSUE THREAD

**Issue by sogun771** [firefly-iii/firefly-iii#7688]
### Support guidelines

- [X] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)

### I've found a bug and checked that ...

- [X] ... [the documentation](https://docs.firefly-iii.org/) does not mention anything about my problem
- [X] ... there are no open or closed issues that are related to my problem
- [X] ... it's [definitely a Firefly III issue, not me](https://github.com/firefly-iii/firefly-iii/blob/main/.github/its_you_not_me.md)

### Description

This issue started to appear once upgraded to 6.0.15 with an error message affecting Firefly III Data Import Tool

![image](https://github.com/firefly-iii/firefly-iii/assets/55146359/426458e1-4277-48ac-a944-b8f80f33b9b2)

This was working well from 6.0.14. 

Not totally sure this is a bug though.

Thank you

PS: Thank you for developing this solution. It is really helpful for managing personal finance. 

### Debug information

Debug information generated at 2023-06-26 11:15:50 Asia/Tokyo for Firefly III version **6.0.15**.

| Scope | Version |
| --- | --- |
| Firefly III | 6.0.15 |
| Firefly III API | 2.0.4 |
| Build | 506, 22-06-2023 03:58:31 UTC |
| Base Build | 10, 21-06-2023 21:55:03 CEST |
| DB version | 19 (exp. 19) |
| PHP | `8.2.7` |
| Host | `Linux` |

| System info | Value |
| --- | --- |
| System TZ | Asia/Tokyo |
| Browser TZ | Asia/Tokyo |
| App environment | Production |
| App debug mode | false |
| App cache driver | redis |
| App logging | notice, stack |
| Display errors | Off |
| Error reporting | ALL errors |
| Max upload | 67108864 (64 MB) |
| Interface | apache2handler |
| Default language | en_US |
| Default locale | equal |
| BCscale | 12 |
| DB drivers | mysql, pgsql, sqlite |
| Current driver | mysql |
| Trusted proxies (.env) | `**` |

| User info | Value |
| --- | --- |
| Login provider | eloquent  |
| User guard | web  |
| Headers | N/A, N/A  |
| Stateful domains |  |
| Session start | 2023-06-01 00:00:00 |
| Session end | 2023-06-30 23:59:59 |
| Session first | 2014-01-01 00:00:00 |
| User ID | 1 |
| User language | en_US |
| User locale | en_US |
| Attempt at "en_US.utf8" | 'en_US.utf8' |
| Attempt at "en_US.UTF-8" | 'en_US.UTF-8' |
| User agent | Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/114.0.0.0 Safari/537.36 Edg/114.0.1823.58 |

### Steps to reproduce

1) upgrade from 6.0.14 to 6.0.15
2) access Firefly III Data Import Tool

**Comment by JC5** [comment]
Mmm, very weird. What happens after you browse to /flush?

**Comment by sogun771** [comment]
Thank you for your reply.

I did try flush without success. I also tried to reauthenticate and start over and even redeploying the .env with same result.

I will try to rollback my version to 6.0.14 to see if the issue is really coming from 6.0.15

**Comment by sogun771** [comment]
Confirmed it's same now with 6.0.14 so must be something from my environment and not a real bug. Will continue investigating :)

**Comment by JC5** [comment]
Thanks, I am curious to learn more as well.

**Comment by JC5** [comment]
If it helps: Firefly III does not name its routes "api/v1/about", the route in question is called "api.v1.about.index": https://github.com/firefly-iii/firefly-iii/blob/main/routes/api.php#L719

**Comment by sogun771** [comment]
I fixed it but still don't get the root cause outside of a cache issue probably (as you possibly suggested with the flush option)

I have redeployed completely my stack using the develop tag which has version up FIDI and it got fixed. Then rolled back to the latest. Must have clean up some logic in the code.

![image](https://github.com/firefly-iii/firefly-iii/assets/55146359/99e9d40f-dc6b-41ab-9f65-68217938f0bb)

If the issue occurred again, I'll try to dig more on it and share if i find something interesting

Thank you for this awesome solution for managing personal finance. :)

If no issue I'll go ahead and close this ticket
